codespell_lib/data/dictionary.txt

```diff
@@ -14478,6 +14575,7 @@ wokr->work
 wokring->working
 wollow->follow, wallow,
 wollowing->following, wallowing,
+womens->women
```

Comment:
The default should be women's surely?

Reply:
I remove it.

Reply:
I think you can leave it in @Gelma , but just add women's as the primary suggested replacement.

Reply:
ok, done.